Updating the Book dash chapter in community handbook (#2062)
alan-turing-institute/the-turing-way: book/website/community-handbook/bookdash.md

(ch-bookdash)=
# _The Turing Way_ Book Dashes

_The Turing Way_ [Book Dash events](https://the-turing-way.netlify.app/community-handbook/bookdash.html) are a less intense version of [Book Sprints](https://en.wikipedia.org/wiki/Book_sprint), where participants collaboratively work on _The Turing Way_ book synchronously to develop new chapters and review/edit existing ones to make them more accessible, comprehensive and up-to-date.

> They also contribute to enhancing the project by improving the ways we work in the community and take lead on accomplishing different tasks or subprojects.

Reviewer: ```suggestion
They also contribute to enhancing the project by improving the ways we work in the community and take the lead on accomplishing different tasks or subprojects.
```

In the past, we have organised 1-1.5 day long Book Dash events in person or partially remote.

> However, with the COVID-19 pandemic situation, we movedthe last two Book Dash's online (November 2020 and May 2021) and designed them with multiple short co-working sessions spread over 5 days for flexible participation by members in different time zones.

Reviewer: ```suggestion
However, with the COVID-19 pandemic situation, we moved the last two Book Dash's online (November 2020 and May 2021) and designed them with multiple short co-working sessions spread over 5 days for flexible participation by members in different time zones.
```

## Inviting Diverse Contributions

As a community-driven guidebook, _The Turing Way_ aims to co-create contents that are comprehensible and beneficial for the wider community of data scientists and researchers.
Hence, _The Turing Way_ specifically welcomes contributors from diverse fields, identities, and backgrounds who can propose ideas and work on new aspects of an existing chapter or create new chapters in its guides.

All the contributions are managed through GitHub.
Since many participants of the Book Dash are new community members, and often new users of GitHub, the team members make sure that they take time to introduce the project and teach collaboration through GitHub before the Book Dash.

> The skills and contributions that we invite at the book dash include, but are not limited to:

Author: ```suggestion
The skills and contributions that we invite at the Book Dash include, but are not limited to:
```

* In the past, due to the funding model of _The Turing Way_, we had to limit our selection to the contributors within European countries.

**For participants attending a Book Dash remotely:** 
* We have experimented with hybrid (partial remote) participation where one of the participants coordinated with their team remotely.
Based on the success of this hybrid format and further demand for more remote participation, we have had two virtual events that did not involve travel.
In the future, we aim to experiment with a multi-hub format allowing multiple small groups across the globe to meet in person as they work synchronously with other international contributors remotely.
* Financial support is made available to support the remote accessibility requirements of the participants at hybrid and fully remote Book Dashes.
It includes (but not be limited to) temporary access to high-speed internet, childcare grant and live transcription during the event.
These bursaries are also available for rent or purchasing small hardware such as headphones or webcam to enhance participants overall experience.

> * There is also financial support for subsistence costs such as buying meals or treat for the online social events during the event.

Reviewer 2: ```suggestion
* There is also financial support for subsistence costs such as buying meals or treat for the online social events during the Book Dash.
```
